Userland/Applets/ClipboardHistory/ClipboardHistoryModel.cpp

```diff
@@ -32,6 +32,8 @@ String ClipboardHistoryModel::column_name(int column) const
         return "Size";
     case Column::Time:
         return "Time";
+    case Column::IsPersistent:
+        return "P";
```

Comment:
I tried since it feels more natural but it actually conflicts with:

```cpp
    enum ClipboardEntryState {
        Persistent,
        Volatile
    };
```

The compiler is not very happy about that :hot_face: